pkg/virt-controller/services/template_test.go

```diff
@@ -2897,6 +2907,37 @@ var _ = Describe("Template", func() {
 			})
 		})
 
+		It("should call sidecar creators", func() {
```

Comment:
This replaces the unit test that were deleted in next commit (virt-controller template tests), right?

Reply:
This is testing the new infra code added to create sidecars.
We are suppose to test the logic that appears in `RenderLaunchManifest`.

The tests remove later are usually specific to a sidecar, something that is no longer relevant in this flow (i.e. this flow no longer cares which sidecar is created).

If you however find that there is a flow which is now not covered due to the move, let me know please.

Reply:
Is there a test that verifies default image is set when no image is registed in Kubevirt config?

Reply:
You mean this one?
https://github.com/kubevirt/kubevirt/pull/10423/files#diff-6ff4f975d10786cfd51dafcf6bd99a615b3a4d1acc34c524bb9fa9d872e2ee30R188